demo/src/storyblok/RichText.astro

```diff
@@ -52,5 +52,10 @@ const { text } = blok;
       props: { blok },
     };
   }}
+  stringResolver={() => {
+    return {
+      component: "span",
+    };
+  }}
```

Comment:
Just an optional idea. we might update the example with the full usage, meaning to take that string from fn prop and return an actual content value in the object.
I would also leave some small comment (since this is example) in this function like:

```js
 stringResolver={(content) => {
    // custom logic to manipulate content string value

    return {
      component: "span",
      content,
    };
  }}
```

or alternatively this could be part of JSDoc comment to showcase the example of possible application of this function.

Reply:
done